tox.ini

```diff
@@ -26,11 +26,11 @@ sphinx_common_args =
   . \
   "{envdir}/docs_out"
 deps =
-  -r{toxinidir}/docs/requirements.txt
+  -r {toxinidir}/docs/requirements.in
```

Comment:
Don't add such spaces, they tend to break things. Also, this change must be coupled with adding `-c`.